antora/supplemental-ui/css/navbar.css

```diff
@@ -1,3 +1,3 @@
 .navbar {
-    background: #085870 !important;
+    background: #0476d0 !important;
```

Comment:
@wmedvede can you try one of these colors and see which one matches better?

> Deep Navy Blue (#002F5F) – A dark blue that matches the logo's darker tones, providing a professional and elegant look.

> Teal Blue (#007E99) – A slightly lighter shade of blue to complement the lighter parts of the logo and create a fresh, modern vibe.

> Charcoal Black (#1C1C1C) – A neutral and bold black that can make the logo stand out while maintaining simplicity.

> Soft Orange (#F4A261) – A muted orange that aligns with the logo’s orange wave, creating a warm and inviting feel.

> Cool Gray (#E0E0E0) – A neutral light gray that contrasts the logo without overshadowing it, ideal for a clean design.

AI suggested it based on our colors.